ci/vars-functional-test.yml

```diff
@@ -6,7 +6,7 @@
 # skip os-net-setup
 cifmw_os_net_setup_config: []
 post_deploy_00_run_functional_test:
-  source: "{{ ansible_user_dir }}/{{ zuul.projects['github.com/infrawatch/feature-verification-tests'].src_dir }}/ci/run_autoscaling_osp18.yml"
+  source: "{{ ansible_user_dir }}/{{ zuul.projects['github.com/infrawatch/feature-verification-tests'].src_dir }}/ci/run_autoscaling_metrics_osp18.yml"
```

Comment:
Are we going to have to rename this every time we add a new test? I think we should call it "run_tests_osp18.yml" or something similar that lists the version.